### ISSUE TITLE
[bug] Conan download from artifactory instance seems not to be working

### ISSUE THREAD

**Issue by timviola-cmi** [conan-io/conan#13466]
### Environment details

* Operating System+version: Windows 10
* Compiler+version: custom embedded compiler
* Conan version: 2.0.1
* Python version: 3.11.0

### Steps to reproduce

run "conan download -r <remote> <package reference>" against an artifactory instance

### Logs

This issue is for all packages. We are unable to download any packages from our artifactory instance, though conan install works fine.

```
C:\Users\xxxxx>conan download -r yyyy package/version
ERROR: 500: org.jfrog.repomd.conan.exceptions.ConanInfoException: Missing section in conaninfo.txt. [Remote: yyyy]
```

**Comment by memsharded** [comment]
Hi @timviola-cmi 

Thanks for your question.
I think that you are hitting a bug/limitation in Artifactory that was fixed in Artifactory 7.52. Please try to upgrade your server and it should go away. (This would be a duplicate of https://github.com/conan-io/conan/issues/12418, I think we could close this issue.)

**Comment by AbrilRBS** [comment]
Closing this as both duplicated and solved, thanks!
